waspc/data/Generator/templates/server/src/core/auth.js

```diff
@@ -48,7 +49,17 @@ export async function getUserFromToken(token) {
     }
   }
 
-  const user = await prisma.{= userEntityLower =}.findUnique({ where: { id: userIdFromToken } })
+  const user = await prisma.{= userEntityLower =}
```

Comment:
It seemed to me that we are storing auth id in JWT token, but here it is user id, what did I get wrong?

Reply:
We are sometimes using `User.id`, sometimes `Auth.id` in order to avoid needless DB calls. Does that make sense? Should we unify those two?

Reply:
Ha hm but wait, then those are not the same JWT tokens, right? Otherwise you wouldn't know which id you are reading -> can you expand a bit on your comment, in which situations do we use which?

Reply:
We use different information for the auth token and the email verification & password reset tokens.

Auth token: uses `User.id`

Email verification & password reset token: uses `email` as of the latest change in the code. This enables us to find the right identity and mark the `email` as confirmed.

Reply:
Ah ok! I forgot we have these different types of tokens. Ok, then it makes sense I believe! Is it clear enough in code when we are dealing with which token -> auth vs email verification vs password reset?

Reply:
Yes, it seems clear enough to me 👍